modules/builds-troubleshooting-service-certificate-generation.adoc

```diff
@@ -7,6 +7,8 @@
 
 If your request for access to resources is denied:
 
+//Annotations - remove?
```

Comment:
Keep - this is not related to the default/override annotations used in the cluster config.